oauth2_http/javatests/com/google/auth/TestUtils.java

```diff
@@ -64,31 +63,35 @@ public class TestUtils {
   public static void assertContainsBearerToken(Map<String, List<String>> metadata, String token) {
     assertNotNull(metadata);
     assertNotNull(token);
-    assertTrue(hasBearerToken(metadata, token), "Bearer token not found");
+    assertTrue("Bearer token not found", hasBearerToken(metadata, token));
   }
 
   public static void assertNotContainsBearerToken(
       Map<String, List<String>> metadata, String token) {
     assertNotNull(metadata);
     assertNotNull(token);
-    assertFalse(hasBearerToken(metadata, token), "Bearer token found");
+    assertTrue("Bearer token found", !hasBearerToken(metadata, token));
```

Comment:
assertFalse and no ! works better here ) easier to validate

 (i know here and other places its a result of revert, but good idea to keep some better nits)